libclamav/regex_suffix.c

```diff
@@ -480,7 +505,7 @@ cl_error_t cli_regex2suffix(const char *pattern, regex_t *preg, suffix_callback
                cli_errmsg("cli_regex2suffix: unable to strdup regex.pattern");
```

Comment:
Also missing a new line `\n` here and in a few other error messages in this file.